wordpress-objects.taxonomy.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+class Taxonomy {
+
+	protected $_tax;
+
+	public function get( $taxonomy ) {
```

Comment:
@mattheu this should be a `static` sorry. And also, I'd say return a `WP_Error` rather than exceptions. I think we should move away from exceptions in this project.